vignettes/comparison.Rmd

```diff
@@ -0,0 +1,322 @@
+---
+title: Exemplary comparison of Bayesian and frequentist MMRM fits
+date: "`r format(Sys.time(), '%d/%m/%Y')`"
+output:
+  rmarkdown::html_document:
+    theme: spacelab
+    highlight: default
+    toc: yes
+    toc_float: yes
+    number_sections: true 
+vignette: >
+  %\VignetteIndexEntry{Exemplary comparison of Bayesian and frequentist MMRM fits}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+```{r setup, eval=TRUE, include=FALSE, warning=FALSE, message=FALSE}
+rm(list = ls())
+library(knitr)
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  prompt = TRUE,
+  comment = NA,
+  size = "scriptsize",
+  fig.height = 5,
+  fig.width = 6,
+  dpi = 320,
+  fig.align = "center",
+  fig.path = "figures/"
```

Comment:
Could the `fig.path` be vignette-specific so it does not conflict with other vignettes?

Reply:
Yes, the path has now been changed to `"comparison_figures/"` (86133299ae6be79a7dc7242fa452c58b474c4eb2).